src/accelerate/utils/dataclasses.py

```diff
@@ -1623,27 +1651,63 @@ class FullyShardedDataParallelPlugin:
     def __post_init__(self):
         from torch.distributed.fsdp import (
             BackwardPrefetch,
-            CPUOffload,
             ShardingStrategy,
         )
 
+        _fsdp2_warnings = {}
```

Comment:
Note this needs to be a `set()` for what you want
```suggestion
        _fsdp2_warnings = set()
```